docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -263,30 +271,30 @@ inputSumAbs: 5
 largest: 22
 ```
 
-Note that when using `Provable.if`, like in a JS ternary, both branches are executed. Unlike normal programming, because SnarkyJS under the hood is creating a zk circuit, there is no primitive for if statements where only one branch is executed.
+Both branches are executed when using `Provable.if`, like in a JavaScript ternary. Because SnarkyJS is creating a zk circuit, there is no primitive for `if` statements where only one branch is executed.
 
 ## Assertions and Constraints
 
 SnarkyJS functions are compiled to generate circuits.
 
-When a transaction is proven in SnarkyJS, SnarkyJS is proving that the program logic is computed according to the written program, and all assertions are holding true.
+When a transaction is proven in SnarkyJS, the proof is that the program logic is computed according to the written program, and all assertions are holding true.
```

Comment:
With this rewrite, we avoid having SnarkyJS do things.
Anthropomorphism guidance (don't do it) https://developers.google.com/style/anthropomorphism 

what other language can we use?